guides/NEW_API.md

```diff
@@ -0,0 +1,45 @@
+Creating a new API microservice
+==================
+
+The following steps will guide you through adding a new API to our platform.
+
+**NOTE:** For the specifics of getting a new service up and running follow our [new app](NEW_APP.md) guide first, and come back here for the specific *additional* steps required for APIs.
+
+Prerequisites
+-------------
+
+If any of the following Github links 404, likely you need to be added to the ONSdigital organisation.
+
+To follow this guide you will need:
+
+* A Github account with access to the ONSdigital organisation
+
+* The [dp-api-router](https://github.com/ONSdigital/dp-api-router) repo cloned locally
+
+* [dp-configs](https://github.com/ONSdigital/dp-configs) repo cloned locally
+
+* The ['magic port'](https://github.com/ONSdigital/dp-setup/blob/develop/PORTS.md) assigned to your microservice API 
+
+Getting started
+---------------
+
+1. Identify the top level route your API will be responsible for. This likely can be taken from the repository name e.g. dp-dataset-api has the top level route **/datasets**. Make sure it doesn't clash with any existing routes [in the router](https://github.com/ONSdigital/dp-api-router/blob/develop/service/service.go#L126)
+
+2. Add a new API URL variable to the dp-api-router [config.go](https://github.com/ONSdigital/dp-api-router/blob/develop/config/config.go#L19) and set a [default value](https://github.com/ONSdigital/dp-api-router/blob/develop/config/config.go#L71) using the localhost:port combination the app will default to when running locally
```

Comment:
It would be worthwhile to add the link - https://github.com/ONSdigital/dp/blob/main/guides/PORTS.md which informs of all the local ports for each app